src/referrals/utils/referral-slug.util.ts

```
@@ -22,22 +22,32 @@ function randomBase36(length: number): string {
-    .replace(/[\u0300-\u036f]/g, '') // diacritics
-    .replace(/\s+/g, '') // remove spaces
-    .replace(/[^a-z0-9_]/g, ''); // remove special chars
-  return s;
+    .replace(/[\u0300-\u036f]/g, '')
+    .replace(/\s+/g, '')
+    .replace(/[^a-z0-9_]/g, '');
+}
+
+// Keep for resolveSlug backward compat \u2014 only trims, no stripping
+export function standardizeSlugInput(input: string): string {
+  return String(input ?? '').trim();
```

ESLint

[error] Replace `''` with `""`

(prettier/prettier)